lib/code_corps/policy/category.ex

```diff
@@ -1,9 +1,7 @@
 defmodule CodeCorps.Policy.Category do
-  alias CodeCorps.User
+  def create?(%{admin: true}), do: true
+  def create?(%{admin: false}), do: false
 
-  def create?(%User{admin: true}), do: true
-  def create?(%User{admin: false}), do: false
-
-  def update?(%User{admin: true}), do: true
-  def update?(%User{admin: false}), do: false
+  def update?(%{admin: true}), do: true
+  def update?(%{admin: false}), do: false
```

Comment:
I understand your goal here is reduction of code, but we tend to prefer using stronger type constraints unless we have to loosen them for some reason. 

In fact, what we'd prefer here is to rewrite these as 

```Elixir
@spec create?(User.t) :: boolean
def create?(%User{admin: true}), do: true
def create?(%User{}), do: false

@spec update?(User.t) :: boolean
def update?(%User{admin: true}), do: true
def update?(%User{}), do: false
```